source/MaterialXGenShader/ColorManagementSystem.cpp

```diff
@@ -88,4 +88,107 @@ ShaderNodePtr ColorManagementSystem::createNode(const ShaderGraph* parent, const
     return shaderNode;
 }
 
+
+void ColorManagementSystem::getPortConnections(ShaderGraph* graph, ShaderNode* colorTransformNode,
```

Comment:
@feldstj : new code to wrap up a transform now with pre / post type conversion.
Reused for both input and ouptut port connections by just returning the input and output shader port.